src/views/DashboardView/VolunteerOnboardingChecklist.vue

```diff
@@ -0,0 +1,203 @@
+<template>
+  <div>
+    <template v-if="!user.isVolunteerApproved">
+      <div>
+        <h3>
+          You need to be approved! To be approved, you need to do the following:
+        </h3>
+        <div class="checklist">
+          <div class="dropdown">
+            <div class="round" v-for="type in checklist" :key="type.index">
+              <div
+                :class="{ active: !user.onboarding[type].submitted }"
+                class="dropdown__header"
+                @click="toggleDropdown($event)"
+              >
+                <input
+                  type="checkbox"
+                  :id="type"
+                  v-model="user.onboarding[type].submitted"
+                  @change="save()"
+                />
```

Comment:
Does this checkbox need to be an interactive control? It seems to me that the process of verifying that required documents have been submitted is restricted to admins only, so it makes no sense to allow non-admin volunteers to change the state of this checkbox.